src/node/coin.cpp

```diff
@@ -12,6 +12,7 @@ void FindCoins(const NodeContext& node, std::map<COutPoint, Coin>& coins)
 {
     assert(node.mempool);
     LOCK2(cs_main, node.mempool->cs);
+    assert(node.chainman);
```

Comment:
would be nice to assert before cs_main, because cs_main "belongs" to the chainman